doubly_linked_list/doubly_linked_list.py

```diff
@@ -58,6 +77,7 @@ def move_to_end(self, node):
     pass
 
   def delete(self, node):
+  
     pass
     
   def get_max(self):
```

Comment:
It looks like you still need to work on move_to_front, move_to_end, delete, and get_max to get these tests to pass. Keep it up!